code/modules/cargo/bounties/atmos.dm

```diff
@@ -0,0 +1,49 @@
+/datum/bounty/item/atmospherics
+	name = "Gas Parent"
+	description = "Shit's broken if you see this."
+	reward = CARGO_CRATE_VALUE * 15
+	wanted_types = list(/obj/item/tank = TRUE)
+	/// How many moles are needed to fufill the bounty?
+	var/moles_required = 20
+	/// Typepath of the gas datum required to fufill the bounty
+	var/gas_type
+
+/datum/bounty/item/atmospherics/applies_to(obj/O)
+	if(!..())
+		return FALSE
+	var/obj/item/tank/T = O
```

Comment:
two var names to improve here: `O` and `T`

Reply:
Resolved